guides/reject_invalids.md

```diff
@@ -216,64 +216,166 @@ protocol bgp PEER1 from PEERS_TEMPLATE {
 }
 ```
 
-## Nokia SR-OS
-
-RPKI validator can be configured within the:
-* Base router
-* Via OOB (example below)
-
-Configure RTR to the RPKI validator(s) via OOB: 
-```
-A:SR-OS# /configure router "management" 
-A:SR-OS>config>router# info 
-----------------------------------------------
-#--------------------------------------------------
-echo "Origin Validation Configuration"
-#--------------------------------------------------
-        origin-validation
-            rpki-session 10.1.1.6
-                description "rtr server"
-                port 323
+## Nokia SR OS
+
+RPKI validator can be configured via:
+* The Base routing instance (example below)
+* The management routing instance on OOB Ethernet port
+
+#### Configure RTR to the RPKI validator(s)
+
+Classic CLI configuration:
+```
+A:br1-nyc>config>router>origin-validation# info detail
+            rpki-session 192.168.1.1
+                description "Routinator RPKI Server"
```

Comment:
no need to promote unsafe software written by people who won't even use their own software for bgp routing! :)

Reply:
Removed...